ras_backstage/authentication/uaa.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+import requests
+from requests import HTTPError
+from json import JSONDecodeError
+from flask import current_app
+
+logger = logging.getLogger(__name__)
+
+
+def request_uaa_public_key(app):
+    headers = {
+        'Accept': 'application/json',
+    }
+
+    public_key_url = f'{app.config["UAA_SERVICE_URL"]}/token_key'
+    response = requests.get(public_key_url, headers=headers)
```

Comment:
This needs to be handled correctly i.e. at least wrap it in a requests.RequestException handler

Reply:
Agree, should read the documentation ;)